frontend/packages/pipelines-plugin/src/components/pipelines/hooks.ts

```diff
@@ -22,8 +22,18 @@ import { metricQueries, PipelineQuery } from './pipeline-metrics/pipeline-metric
 
 type Match = RMatch<{ url: string }>;
 
-export const usePipelinesBreadcrumbsFor = (kindObj: K8sKind, match: Match) =>
-  useTabbedTableBreadcrumbsFor(kindObj, match, 'pipelines', pipelinesTab(kindObj));
+export const usePipelinesBreadcrumbsFor = (kindObj: K8sKind, match: Match) => {
+  const isAdminPerspective = useActivePerspective()[0] === 'admin';
+  const navOption = isAdminPerspective ? `pipelines` : 'dev-pipelines';
+  return useTabbedTableBreadcrumbsFor(
```

Comment:
PipelineRuns details page breadcrumbs is taking us back to `dev-pipelines` route, it should redirect to default `k8s/ns/viraj/tekton.dev~v1beta1~PipelineRun` route
![image](https://user-images.githubusercontent.com/9964343/129560734-8ba81a4e-dc95-4d4a-b677-9a550751b521.png)